src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -99,4 +126,19 @@ private function removeMessages(string $failureTransportName, array $ids, Receiv
             }
         }
     }
+
+    private function removeAllMessages(ListableReceiverInterface $receiver, SymfonyStyle $io, bool $shouldDisplayMessages): void
```

Comment:
we don't need a ListableReceiverInterface here, do we? we should accept a simple ReceiverInterface then

Reply:
We do, to be sure we access the `all()` method of `$receiver`

Reply:
what about using get in a loop? would that work?

Reply:
It doesn't seem to work unfortunately. Using `get()` removes the message from the transport after the call, thus the message isn't present anymore on the next command run